.eslintrc.js

```diff
@@ -1,139 +1,115 @@
-/* eslint-disable quote-props */
 module.exports = {
-  'plugins': ['lodash-fp', 'simple-import-sort'],
-  'extends': 'react-app',
-  'settings': {
+  extends: ['airbnb-typescript-prettier'],
```

Comment:
The TypeScript rules included here aren't sourced from AirBnB's style guide, they're from `@typescript-eslint/recommended` plus some additional rules added by this 3rd party package.

Thoughts on depending directly on AirBnB's package and adding `@typescript-eslint/recommended` in our own configuration?

Reply:
I'd prefer sticking with this, only because it's used(visibly) by over a thousand other repositories on github, and the main goal here was simply to have a standard to follow that we won't need to maintain ourselves.

Reply:
I can get behind @trock-broadinst 's thinking on this one :)

Reply:
To be clear, I'm not suggesting configure individual TypeScript related rules. I'm asking about extending from `@typescript-eslint/recommended`, but cutting out the middle man of `eslint-config-airbnb-typescript-prettier`. The argument for basing on AirBnB's style guide as a widely adopted standard doesn't hold as much when we're basing on someone else's config that itself is based on AirBnB's and has a fraction of the adoption.

Reply:
I'm not going to request changes over this.

Reply:
@trock-broadinst you are also going to need a reviewer from data-explorer-eng (CODEOWNERS requirement) in order to merge.

Reply:
@s-rubenstein from `data-explorer-eng` has reviewed this PR